lib/pwwka/receiver.rb

```diff
@@ -79,5 +79,10 @@ def test_teardown
       channel_connector.connection_close
     end
 
+    def parse_payload(payload)
+      return payload if Pwwka.configuration.receive_raw_payload
+      ActiveSupport::HashWithIndifferentAccess.new(JSON.parse(payload))
```

Comment:
Out of scope for this PR, but it looks like this is the only place that `ActiveSupport` is used in this gem.  That's a heavy dependency for this one use-case.  Are there other places we're relying on it that I'm missing?

Reply:
This uses other features of active support and has since inception.  This gem was not designed to use outside of Rails, though it can be if someone is OK with active support.